public/js/length_visualisation.js

```diff
@@ -0,0 +1,339 @@
+import _ from 'underscore';
+import VisualisationHelpers from './visualisation_helpers';
+
+export default class LengthVisualisation {
+  constructor(query, svg_container, algorithm) {
+    this._svg_container_d3 =  d3.select(svg_container[0]);
+    this._svg = {};
```

Comment:
`this._svg` is an object with only one property (`d3` - L36). And `this._svg.d3` is same as `this._svg_container_d3`. Right? Can we have just one `this.svg` instead?